### ISSUE TITLE
gradle build fails for Minecraft 1.20.1 and MinecraftForge 47.0.43 with jarJar enabled

### ISSUE THREAD

**Issue by GeheimagentNr1** [MinecraftForge/ForgeGradle#930]
**Minecraft Version:** 1.20.1

**Forge Version:** 47.0.43

**Description of issue:**
On executing "gradlew build" the following error occures:
>Could not determine the dependencies of task ':jarJar'.
> > java.util.NoSuchElementException (no error message)

**Comment by GeheimagentNr1** [comment]
Log and gradle files:
[Issue.zip](https://github.com/MinecraftForge/MinecraftForge/files/11952244/Issue.zip)

**Comment by GeheimagentNr1** [comment]
I found the issue on my side.
I use a template for my build.gradle, gradle.properties etc. and I had by accident the line "archiveClassifier = 'slim'" for the jar task also in the MinecraftForgeAPI project and because of that, the Minecraft-1.20.1-1.0.0.jar could not be found, because there were only a Minecraft-1.20.1-1.0.0-slim.jar and a Minecraft-1.20.1-1.0.0-dev.jar file.

I removed the "archiveClassifier = 'slim'" line from the jar task in the MinecraftForgeAPI project, and now everything is working fine with the build.

Sorry for bothering and wasting your time.
You are doing a great job.
